src/main/resources/static/templates/layout.html

```diff
@@ -16,13 +16,18 @@
 <!DOCTYPE html>
 <html th:fragment="layout(page,title,description,links,content)" xmlns:th="http://www.thymeleaf.org">
 <head>
-    <!-- Google Tag Manager -->
-    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-            new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-    })(window,document,'script','dataLayer','GTM-N4T4SNK');</script>
-    <!-- End Google Tag Manager -->
+    <!-- OneTrust Cookie Consent -->
+    <meta content='60a709d9-50a1-442d-8f59-072e08a3ddb4' name='onetrust-data-domain'>
+    <meta content='https://tags.tiqcdn.com/utag/vmware/microsites-privacy/prod/utag.js' name='microsites-utag'>
+    <script src="https://code.jquery.com/jquery-2.1.4.js"></script>
+    <script src='//www.vmware.com/files/templates/inc/utag_data.js'></script>
```

Comment:
missing `https:` before the `//` here and on line 23?

Reply:
also, these scripts seem like they might be heavily `vmware.com` oriented. can you confirm they're supposed to work on other domains?